src/opensubtitlescom/cli.py

```diff
@@ -0,0 +1,136 @@
+#!/usr/bin/env python3
+
+import typing as t
+from pathlib import Path
+import json
+import sys
+import argparse
+import os
+import logging
+
+from opensubtitlescom import OpenSubtitles, FileUtils
+
+log = logging.getLogger(__name__)
+
+
+APP_NAME = "CLI Test"
+APP_VER = "0.0.0"
+API_KEY = "F0f1dadQ89xKIP5TIsu3Y8KT7TiDBIfG"
```

Comment:
If this is your API key, I recommend deleting this pull request to prevent it from being stored in the Git history.